pkg/schedule/schedulers/scatter_range.go

```diff
@@ -90,7 +90,7 @@ func (conf *scatterRangeSchedulerConfig) getEndKey() []byte {
 func (conf *scatterRangeSchedulerConfig) getSchedulerName() string {
 	conf.RLock()
 	defer conf.RUnlock()
-	return fmt.Sprintf("scatter-range-%s", conf.RangeName)
+	return fmt.Sprintf("%s-%s", types.ScatterRangeScheduler, conf.RangeName)
```

Comment:
need to consider the compatibility issue

Reply:
https://github.com/tikv/pd/pull/8607/commits/7f1434335bb5f623cc8aa6041aabb45eaee48be7

This commit fixed it. You can check it in its comment.

Reply:
friendly ping